share/revng/test/tests/analysis/CollectCFG/x86_64/switch-jump-table.S.cfg.yml

```diff
@@ -3,44 +3,80 @@
 #
 
 ---
-- FunctionMetadata:
-    ControlFlowGraph:
-      - End: "2"
-        ID: "1"
-        Successors:
-          - Destination: "2"
-            Type: DirectBranch
-          - Destination: "6"
-            Type: DirectBranch
-      - End: "3"
-        ID: "2"
-        Successors:
-          - Destination: "3"
-            Type: DirectBranch
-          - Destination: "4"
-            Type: DirectBranch
-          - Destination: "5"
-            Type: DirectBranch
-          - Destination: "6"
-            Type: DirectBranch
-      - End: "4"
-        ID: "3"
-        Successors:
-          - Destination: ":Invalid"
-            Type: Return
-      - End: "5"
-        ID: "4"
-        Successors:
-          - Destination: ":Invalid"
-            Type: Return
-      - End: "6"
-        ID: "5"
-        Successors:
-          - Destination: ":Invalid"
-            Type: Return
-      - End: "7"
-        ID: "6"
-        Successors:
-          - Destination: ":Invalid"
-            Type: Return
-  OriginalName: _start
+FunctionMetadata:
```

Comment:
1. Does this actually work?
   The original file was a list, you might be testing nothing. Try to introduce an error and make sure the test fails.
2. You seem to have committed the output of the tool, but you have to prune everything is not strictly necessary. See the original file. For instance, `Kind:` and `InlinedFrom:`. Please review carefully the differences.

Reply:
Yep, it fails if there was an error. OK. Simplified.